server_admin/topics/roles/role-scope-mappings.adoc

```diff
@@ -2,30 +2,16 @@
 
 === Role Scope Mappings
 
-When an OIDC access token or SAML assertion is created, all the user role mappings of the user are, by default, added as claims
-within the token or assertion.  Applications use this information to make access decisions on the resources controlled by that
-application.  In {project_name}, access tokens are digitally signed and can actually be re-used by the application
-to invoke on other remotely secured REST services.  This means that if an application gets compromised or there is a rogue
-client registered with the realm, attackers can get access tokens that have a broad range of permissions and your whole
-network is compromised.  This is where _role scope mappings_ becomes important.
+On creation of an OIDC access token or SAML assertion, the user role mappings become claims within the token or assertion.  Applications use these claims to make access decisions on the resources controlled by the application.  {project_name} digitally signs access tokens and applications re-use them to invoke remotely secured REST services.  There is a danger that an attacker can obtain these tokens and use their permissions to compromise your networks. _Role Scope Mappings_ are available to prevent this.
```

Comment:
Instead of "There is a danger that an attacker.." 
How about "However, these tokens have an associated risk. An attacker..

for "_Role Scope Mappings_ are available to prevent this."
How about "To prevent this situation, use _Role Scope Mappings_."